functions.php

```diff
@@ -524,16 +524,52 @@ function twenty_twenty_one_non_latin_languages() {
  * @return void
  */
 function twentytwentyone_customize_preview_init() {
+	wp_enqueue_script(
+		'twentytwentyone-customize-helpers',
+		get_theme_file_uri( '/assets/js/customize-helpers.js' ),
+		array(),
+		get_theme_file_path( 'assets/js/customize-helpers.js' ),
```

Comment:
This should be a version, not a file path— looks like the other scripts use `wp_get_theme()->get( 'Version' ),`

Reply:
fixed